src/styles/_card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 20px;
+  padding: 32px 16px 17px;
+  color: $main-color;
+  font-family: $font-family;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  margin-left: 3px;
+}
+
+.product-name {
+  font-size: $regular-font-size;
+  font-weight: $regular-font-weight;
+  line-height: $regular-line-height;
+  margin-top: 36px;
+  margin-bottom: 0;
```

Comment:
you can reset default margin for all `p`

Reply:
not fixed
https://github.com/mate-academy/layout_product-cards/pull/3764/files#r1393214339